### PR TITLE
docs: router doc updates and add IE polyfills to TS samples

### PR DESCRIPTION
**Pull request by wardbell**

### CODE REVIEW

**Comment by PascalPrecht** [comment]
This LGTM

File: public/docs/ts/latest/guide/router.jade [hunk 1]
```diff
@@ -287,9 +289,9 @@ figure.image-display
   "HTML 5 style" URLs.
 
   We must **add a [&lt;base href> element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) tag**
-  to the `index.html` to make this work.
-  The `href` value specifies the base URL to use for all *relative* URLs within a document including
-  links to css files, scripts, and images.
+  to the `index.html` to make `pushState` routing work. 
+  The browser also needs the base `href` value to prefix *relative* URLs when downloading and linking to 
+  css files, scripts, and image.
```
Comment (PascalPrecht) [inline public/docs/ts/latest/guide/router.jade]: image -> images

**Comment by PascalPrecht** [comment]
Ah, found one thing to fix. Otherwise LGTM

File: public/docs/ts/latest/guide/router.jade (hunk 1 above)
Comment (wardbell) [inline public/docs/ts/latest/guide/router.jade]: Thx. Fixing. Also adding a note about IE!

**Comment by wardbell** [comment]
Yikes!  Thanks Eric!

**Pull request closed without merging**